<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupWindow</class>
 <widget class="QMainWindow" name="SetupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SetupWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="exptSetupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>541</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Experiment Setup</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="exptNoLayout">
      <item>
       <widget class="QLabel" name="exptNoLabel">
        <property name="text">
         <string>Experiment No:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="exptNoText">
        <property name="text">
         <string>EXP-21-BP000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exptNoSetButton">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exptNoClearButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="positionsInputLayout">
      <item>
       <widget class="QLabel" name="positionsInputLabel">
        <property name="text">
         <string>Get Positions from:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fromFile">
        <property name="text">
         <string>File</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fromMicroManager">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Micromanager</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="fileSelectionLayout">
      <item>
       <widget class="QLabel" name="fileSelectionLabel">
        <property name="text">
         <string>If File Select File:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fileSelectionButton">
        <property name="text">
         <string>Select File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="eventsCreationLayout">
      <item>
       <widget class="QLabel" name="eventsCreationLabel">
        <property name="text">
         <string>Events Creation:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eventsCreationButton">
        <property name="text">
         <string>Create Events</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="validateExptSetupButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>200</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Validate Experiment Setup</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="analysisSetupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>541</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Analysis Setup</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="segmentationLayout">
      <item>
       <widget class="QLabel" name="segmentationLabel">
        <property name="text">
         <string>Cell Segmentation: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="selectNet">
        <item>
         <property name="text">
          <string>Normal U-net</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Small U-net</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="channelSegLayout">
      <item>
       <widget class="QCheckBox" name="segChannels">
        <property name="text">
         <string>Channel Segmentation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="cellAnalysisLayout">
      <item>
       <widget class="QCheckBox" name="calcDeadAlive">
        <property name="text">
         <string>DeadAlive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="calcGrowthRates">
        <property name="text">
         <string>Growth Rates</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="validateAnalysisSetupButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>170</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Validate Analysis Setup</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="exptSaveCloseButtons">
     <item>
      <widget class="QPushButton" name="resetExptSetupButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeExptSetupButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
